### DISCUSSION TITLE
Edge function deploy failed

### DISCUSSION THREAD

**Discussion by westlinkin** [supabase/supabase#10768] [Questions]
Created and Tested a newly created Function according to [doc](https://supabase.com/docs/guides/functions/quickstart).

Test successfully locally. But when I deploy this function using `supabase --debug functions deploy XXXXXX`:

```
Bundling pre_load_answers
Error: Error bundling function: signal: trace/BPT trap
file:///src/import_map.json
file:///src/index.ts
assertion failed [block != nullptr]: BasicBlock requested for unrecognized address
(BuilderBase.h:550 block_for_offset)
```

99.9% of the time it will fail and show the above message. I only deploy successfully once by accident (the same command line, I don't know why it deployed). 

How to solve this? Does it have something to do with my internet connection? ( I am in China, and I cannot remember whether I was on VPN or not the time it successfully deployed)

Thanks in advance.

**Comment by twgonzalez** [comment]
I am having the exact same issue.  Not sure if it is related to being on macOs 12.4 with M1 MAX?   Does anyone have a solution for this?

**Reply by westlinkin** [reply]
check this out:  https://github.com/supabase/cli/issues/673

**Reply by twgonzalez** [reply]
Thanks!   That resolves it.
